MaskRCNN_no_batch_convergence/model_fpn.py

```diff
@@ -401,6 +472,7 @@ def generate_fpn_proposals_batch_tf_op(multilevel_box_logits, multilevel_label_l
                 # print(anchor_list)
                 anchors = tf.stack(anchor_list)
 
+                """
```

Comment:
Shouldnt' we be feeding `single_level_anchors_boxes` to `generate_bounding_box_proposals` below?

Reply:
Yes, we definitely should

Reply:
Fixed

Reply:
I assume the convergence test had it the correct way?

Reply:
Yeah. I had to fix merge conflicts when changing the PR to be on master and I think that's what introduced this problem. But I'll kick off another run as soon as its merged in just to make sure.